### DISCUSSION TITLE
Welcome to pollingus Discussions!

### DISCUSSION THREAD

**Discussion by SanAlexis** [nyx-ei/pollingus#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by yvankamdem** [comment]
I am Yvan kamdem passionate about new information and communication technologies. currently in perpetual quest for knowledge and that is the reason why I joined the community. I hope to be able to learn and also contribute on the projects.
